docs/alincome/expenses.md

```diff
@@ -0,0 +1,271 @@
+---
+id: expenses
+title: |
+    Expenses: ALExpense and ALExpenseList
+sidebar_label: |
+  Expenses
+slug: /alincome/expenses
+---
+
+The `ALExpenseList` is a class that lets you ask a user about their different expenses in a simple way.
```

Comment:
I've noticed you've used the word 'simple' a couple times. You were trying to avoid that, right?

Reply:
Good catch. I was trying to avoid using it to imply that some coding process was simple, like "Simply change the docassemble configuration or something".  Making things simple (and straight-forward) for our users is an important goal of our libraries though, so it feels valid here? Open to suggestions, but can think of any improvements myself that don't make this sentence longer than it needs to be.

Reply:
"Simple" here seems a bit over confident, but I see what you're saying. I thought I caught a use of the word lower down being used to express that something was simple to do. I'll have to try and track that down.

```suggestion
The `ALExpenseList` is a class that helps you ask a user about their different expenses.
```